cmd/lotus-shed/verifreg.go

```diff
@@ -409,3 +413,115 @@ var verifRegCheckVerifierCmd = &cli.Command{
 		return nil
 	},
 }
+
+var verifRegRemoveVerifiedClientDataCapCmd = &cli.Command{
+	Name:      "remove-verified-client-data-cap",
+	Usage:     "Remove data cap from verified client",
+	ArgsUsage: "<message sender> <verified client ID> <allowance to remove> <verifier 1 ID> <verifier 1 signature> <verifier 2 ID> <verifier 2 signature>",
+	Action: func(cctx *cli.Context) error {
+		if cctx.Args().Len() != 7 {
+			return fmt.Errorf("must specify seven arguments: sender, client, allowance to remove, verifier 1 ID, verifier 1 signature, verifier 2 ID, verifier 2 signature")
+		}
+
+		srv, err := lcli.GetFullNodeServices(cctx)
+		if err != nil {
+			return err
+		}
+		defer srv.Close() //nolint:errcheck
+
+		api := srv.FullNodeAPI()
+		ctx := lcli.ReqContext(cctx)
+
+		sender, err := address.NewFromString(cctx.Args().Get(0))
+		if err != nil {
+			return err
+		}
+
+		client, err := address.NewFromString(cctx.Args().Get(1))
+		if err != nil {
+			return err
+		}
+
+		allowanceToRemove, err := types.BigFromString(cctx.Args().Get(2))
+		if err != nil {
+			return err
+		}
+
+		verifier1Addr, err := address.NewFromString(cctx.Args().Get(3))
+		if err != nil {
+			return err
+		}
+
+		verifier1Sig, err := hex.DecodeString(cctx.Args().Get(4))
+		if err != nil {
+			return err
+		}
+
+		verifier2Addr, err := address.NewFromString(cctx.Args().Get(5))
+		if err != nil {
+			return err
+		}
+
+		verifier2Sig, err := hex.DecodeString(cctx.Args().Get(6))
+		if err != nil {
+			return err
+		}
+
+		var sig1 crypto.Signature
+		if err := sig1.UnmarshalBinary(verifier1Sig); err != nil {
+			return xerrors.Errorf("couldn't unmarshal sig: %w", err)
+		}
+
+		var sig2 crypto.Signature
+		if err := sig2.UnmarshalBinary(verifier2Sig); err != nil {
+			return xerrors.Errorf("couldn't unmarshal sig: %w", err)
+		}
+
+		params, err := actors.SerializeParams(&verifreg.RemoveDataCapParams{
+			VerifiedClientToRemove: client,
+			DataCapAmountToRemove:  allowanceToRemove,
+			VerifierRequest1: verifreg.RemoveDataCapRequest{
+				Verifier:          verifier1Addr,
+				VerifierSignature: sig1,
+			},
+			VerifierRequest2: verifreg.RemoveDataCapRequest{
+				Verifier:          verifier2Addr,
+				VerifierSignature: sig2,
+			},
+		})
+		if err != nil {
+			return err
+		}
+
+		vrk, err := api.StateVerifiedRegistryRootKey(ctx, types.EmptyTSK)
```

Comment:
let's `StateGetActor` this, load the multisig state, call `Signers()`, and  confirm that `sender` is one of the signers.